src/main/java/com/solvd/android/Category.java

```diff
@@ -0,0 +1,27 @@
+package com.solvd.android;
+
+import com.zebrunner.carina.webdriver.decorator.ExtendedWebElement;
+import com.zebrunner.carina.webdriver.gui.AbstractUIObject;
+import org.openqa.selenium.SearchContext;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.FindBy;
+
+public class Category extends AbstractUIObject {
```

Comment:
as you decided to make it as component, create please separate folder for components